Agreements/Internal/TCCOW_25.txt

```diff
@@ -0,0 +1,3 @@
+COPX:EGNX:*:CPT:*:*:London TC COWLY:London S25:22000:*:^CPT
+
+COPX:EGNX:*:SILVA:*:*:London TC COWLY:London S25:22000:*:^SILVA
```

Comment:
Need to discuss how this is going to work given we aren't delegating airspace over to Dover in this area - will either need guest ownership, or to make the agreement FL210